### PR TITLE
Add missing google-beta provider to required_providers

### PR DESCRIPTION
**Pull request by amir-hadi**
You are using the google-beta provider in `main.tf`, but it's not declared in the `required_providers` block, which creates a warning on each run.

### CODE REVIEW

**Comment by comment-bot-dev** [comment]
@amir-hadi
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

File: modules/postgresql/versions.tf [hunk 1]
```diff
@@ -29,6 +29,10 @@ terraform {
       source  = "hashicorp/google"
       version = ">= 4.4.0, < 5.0"
     }
+    google-beta = {
+      source  = "hashicorp/google-beta"
+      version = ">= 4.4.0, < 5.0"
+    }
   }
 
   provider_meta "google" {
```
Comment (bharathkkb) [inline modules/postgresql/versions.tf:38]: Could we also add provider_meta for the beta provider?
https://github.com/terraform-google-modules/terraform-google-sql-db/blob/704be5d94eb2e321aabe2c704a6610d569e0bbe2/modules/mssql/versions.tf#L27-L29

**Review by bharathkkb — COMMENTED** [review]
Thanks for the PR @amir-hadi 
Looks like we are also missing it for mysql module https://github.com/terraform-google-modules/terraform-google-sql-db/blob/master/modules/mysql/versions.tf

**Comment by amir-hadi** [comment]
@bharathkkb I also added it for mySQL now.

**Comment by comment-bot-dev** [comment]
@amir-hadi
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Comment by comment-bot-dev** [comment]
@amir-hadi
Thanks for the PR! 🚀
Unfortunately it looks like some of our CI checks failed. See the [Contributing Guide](../blob/master/CONTRIBUTING.md) for details.
- ⚠️check_terraform
 Failed Terraform check. More details below. 
 ```bash 
Running terraform fmt
Running terraform validate
terraform_validate ./examples/mssql-public
Success!
 The configuration is valid.
terraform_validate ./examples/mysql-ha
There are some problems with the configuration, described below.
The Terraform configuration must be valid before initialization so that
Terraform can determine which modules and providers need to be installed.
╷
│
 
Error: 
Duplicate provider_meta block
│
 
│
 
  on ../../modules/mysql/versions.tf line 41, in terraform:
│
 
  41:   
provider_meta "google-beta"
 {
│
 
│
 
A provider_meta block for provider "google" was already declared at
│
 
../../modules/mysql/versions.tf:38,3-25. Providers may only have one
│
 
provider_meta block per module.
╵
terraform_validate ./examples/mysql-private
There are some problems with the configuration, described below.
The Terraform configuration must be valid before initialization so that
Terraform can determine which modules and providers need to be installed.
╷
│
 
Error: 
Duplicate provider_meta block
│
 
│
 
  on ../../modules/mysql/versions.tf line 41, in terraform:
│
 
  41:   
provider_meta "google-beta"
 {
│
 
│
 
A provider_meta block for provider "google" was already declared at
│
 
../../modules/mysql/versions.tf:38,3-25. Providers may only have one
│
 
provider_meta block per module.
╵
terraform_validate ./examples/mysql-public
There are some problems with the configuration, described below.
The Terraform configuration must be valid before initialization so that
Terraform can determine which modules and providers need to be installed.
╷
│
 
Error: 
Duplicate provider_meta block
│
 
│
 
  on ../../modules/mysql/versions.tf line 41, in terraform:
│
 
  41:   
provider_meta "google-beta"
 {
│
 
│
 
A provider_meta block for provider "google" was already declared at
│
 
../../modules/mysql/versions.tf:38,3-25. Providers may only have one
│
 
provider_meta block per module.
╵
terraform_validate ./examples/postgresql-ha
Success!
 The configuration is valid.
terraform_validate ./examples/postgresql-public
Success!
 The configuration is valid.
terraform_validate ./examples/postgresql-public-iam
Success!
 The configuration is valid.
terraform_validate ./modules/mssql
Success!
 The configuration is valid.
terraform_validate ./modules/mysql
There are some problems with the configuration, described below.
The Terraform configuration must be valid before initialization so that
Terraform can determine which modules and providers need to be installed.
╷
│
 
Error: 
Duplicate provider_meta block
│
 
│
 
  on versions.tf line 41, in terraform:
│
 
  41:   
provider_meta "google-beta"
 {
│
 
│
 
A provider_meta block for provider "google" was already declared at
│
 
versions.tf:38,3-25. Providers may only have one provider_meta block per
│
 
module.
╵
terraform_validate ./modules/postgresql
Success!
 The configuration is valid.
terraform_validate ./modules/private_service_access
Success!
 The configuration is valid.
terraform_validate ./modules/safer_mysql
There are some problems with the configuration, described below.
The Terraform configuration must be valid before initialization so that
Terraform can determine which modules and providers need to be installed.
╷
│
 
Error: 
Duplicate provider_meta block
│
 
│
 
  on ../mysql/versions.tf line 41, in terraform:
│
 
  41:   
provider_meta "google-beta"
 {
│
 
│
 
A provider_meta block for provider "google" was already declared at
│
 
../mysql/versions.tf:38,3-25. Providers may only have one provider_meta
│
 
block per module.
╵
terraform_validate ./test/fixtures/mssql-ha
Success!
 The configuration is valid.
terraform_validate ./test/fixtures/mssql-public
Success!
 The configuration is valid.
terraform_validate ./test/fixtures/mysql-ha
There are some problems with the configuration, described below.
The Terraform configuration must be valid before initialization so that
Terraform can determine which modules and providers need to be installed.
╷
│
 
Error: 
Duplicate provider_meta block
│
 
│
 
  on ../../../modules/mysql/versions.tf line 41, in terraform:
│
 
  41:   
provider_meta "google-beta"
 {
│
 
│
 
A provider_meta block for provider "google" was already declared at
│
 
../../../modules/mysql/versions.tf:38,3-25. Providers may only have one
│
 
provider_meta block per module.
╵
terraform_validate ./test/fixtures/mysql-private
There are some problems with the configuration, described below.
The Terraform configuration must be valid before initialization so that
Terraform can determine which modules and providers need to be installed.
╷
│
 
Error: 
Duplicate provider_meta block
│
 
│
 
  on ../../../modules/mysql/versions.tf line 41, in terraform:
│
 
  41:   
provider_meta "google-beta"
 {
│
 
│
 
A provider_meta block for provider "google" was already declared at
│
 
../../../modules/mysql/versions.tf:38,3-25. Providers may only have one
│
 
provider_meta block per module.
╵
terraform_validate ./test/fixtures/mysql-public
There are some problems with the configuration, described below.
The Terraform configuration must be valid before initialization so that
Terraform can determine which modules and providers need to be installed.
╷
│
 
Error: 
Duplicate provider_meta block
│
 
│
 
  on ../../../modules/mysql/versions.tf line 41, in terraform:
│
 
  41:   
provider_meta "google-beta"
 {
│
 
│
 
A provider_meta block for provider "google" was already declared at
│
 
../../../modules/mysql/versions.tf:38,3-25. Providers may only have one
│
 
provider_meta block per module.
╵
terraform_validate ./test/fixtures/postgresql-ha
Success!
 The configuration is valid.
terraform_validate ./test/fixtures/postgresql-public
Success!
 The configuration is valid.
terraform_validate ./test/fixtures/postgresql-public-iam
Success!
 The configuration is valid.
terraform_validate ./test/setup
Success!
 The configuration is valid.
 ```

**Comment by comment-bot-dev** [comment]
@amir-hadi
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Review by bharathkkb — APPROVED** [review]

**Pull request merged**